app/main/views.py

```diff
@@ -3,6 +3,13 @@
 from . import main
 from ..lib.authentication import requires_authentication
 from ..models import Service
+from flask import abort
+from flask import make_response
+from flask import render_template
+from flask import request
+
+from app import services
+from app.services import g6importService
```

Comment:
These imports are a bit wonkey. In other cases we import the function / class from the module rather than just the module. So, for consistency, I'd recommend the second import style. This will mean changing line https://github.com/alphagov/digitalmarketplace-api/pull/9/files#diff-8b9538e63b0d5c7fa84a61f3b8e7ec6dR80